openfecli/tests/commands/test_quickrun.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+import click
+import importlib
+from click.testing import CliRunner
+
+from openfecli.commands.quickrun import quickrun
+
+def test_quickrun():
+    runner = CliRunner()
```

Comment:
I'm guessing this is doing the same as calling the cli via subprocess?

Reply:
I'm not sure if it internally splits off a subprocess, but it is similar enough -- catches stdout, stderr, and return code -- so it gives us what we need. It also handles tempdir, etc. in the `isolated_filesystem` context manager.

[This is the recommended way to test `click` applications.](https://click.palletsprojects.com/en/8.1.x/testing/)